### PR TITLE
Card

### PR DESCRIPTION
**Pull request by TomChapmanGov**

### CODE REVIEW

**Review by jadmnsaad — CHANGES_REQUESTED** [review]
We can leverage built in MUI Grid props for some CSS properties used:

https://mui.com/material-ui/react-grid/

**Review by dinesh-aot — APPROVED** [review]

File: epictrack-web/src/components/myWorkplans/Card/CardBody.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  return (
+    <Grid
+      container
+      direction="column"
+      justifyContent="flex-start"
+      sx={{
+        backgroundColor: Palette.white,
+        padding: "16px 24px",
+      }}
+      gap={1}
```
Comment (jadmnsaad) [inline epictrack-web/src/components/myWorkplans/Card/CardBody.tsx:21]: one more ;p spacing={1}

File: epictrack-web/src/components/myWorkplans/Card/CardBody.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        sx={{ paddingBottom: "8px" }}
+      >
+        <Grid item sx={{ marginTop: "2px" }}>
+          <DotIcon />
+        </Grid>
+        <Grid item>
+          <ETCaption2
+            bold
+            color={Palette.neutral.main}
+            sx={{ fontSize: "14px" }}
```
Comment (jadmnsaad) [inline epictrack-web/src/components/myWorkplans/Card/CardBody.tsx]: when it comes to text properties like fontSize or lineHeight we shouldn't do spot changes for them. There should be a text with fontSize 14px in the shared components if there isn't we should talk to Maria and create one

File: epictrack-web/src/components/myWorkplans/Card/CardBody.tsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+          </ETCaption2>
+        </Grid>
+        <Grid item sx={{ marginTop: "2px" }}>
+          <ClockIcon />
+        </Grid>
+        <Grid item>
+          <ETCaption2
+            bold
+            color={Palette.neutral.main}
+            sx={{ fontSize: "14px" }}
```
Comment (jadmnsaad) [inline epictrack-web/src/components/myWorkplans/Card/CardBody.tsx]: no spot changes for fontSize

File: epictrack-web/src/components/myWorkplans/Card/CardBody.tsx [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+          >
+            46/90 days left
+          </ETCaption2>
+        </Grid>
+      </Grid>
+      <Grid item container direction="row" spacing={1}>
+        <Grid item>
+          <ETCaption1
+            color={Palette.neutral.main}
+            sx={{ fontSize: "13px", letterSpacing: "0.39px" }}
```
Comment (jadmnsaad) [inline epictrack-web/src/components/myWorkplans/Card/CardBody.tsx]: no spot changes for fontSize and letterSpacing, should exist in shared components and if not we should talk to Maria first then add it

File: epictrack-web/src/components/myWorkplans/Card/CardBody.tsx (hunk 1 above)
Comment (TomChapmanGov) [reply epictrack-web/src/components/myWorkplans/Card/CardBody.tsx:21]: i had to leave this as gap, spacing breaks it

**Review by jadmnsaad — CHANGES_REQUESTED** [review]
Noticed spot changes for text properties like fontSize, applying it to any shared text component e.g. ETCaption1 collides with the purpose of having these properties already defined in the reusable component

**Review by jadmnsaad — APPROVED** [review]

**Pull request merged**